src/jobBundle/Controller/JobController.php

```diff
@@ -174,37 +194,79 @@ public function getJobsForUpdate()
      */
     public function updateJob($job_id,Request $request)
     {
-         $jobs = new jobEntity();  
-         $em = $this->getDoctrine()->getManager();
+        if (!$this->jobService->Updatejob($job_id)) {
+            
+           return new Response('There is no such an ID '.$job_id );
+           
+        } else {
+            
          $jobs = $this->getDoctrine()
            ->getRepository(jobEntity::class)
            ->find($job_id);
-            
-       if (!$jobs) {
-           return new Response('Nincs ilyen id, hogy '.$job_id );
-        } else {
-      
+         
         $form = $this->createFormBuilder()
            ->add('name', TextType::class)
            ->add('Update', SubmitType::class)
            ->getForm();
   
-        $name = $form['name']->setData($jobs->getName());
+        $form['name']->setData($jobs->getName());
         
         $form->handleRequest($request);
-        
-            if($form->isSubmitted() && $form->isValid()){
+ 
+            if($this->formProcess3($form,$jobs,$job_id)){
 
-            $name = $form['name']->getData();
-            $jobs->setName($name);
-            $em->flush();
+               $this->addFlash(
+                  'notice',
+                  'Job found'
+               );
+                return $this->redirectToRoute("job_job_getjobs");
+           } 
+           else 
+           {
+                return $this->render('job/job_update_id.html.twig', array(
+                    'form' => $form->createView()
+                 ));
+           }
+        }
+    } 
+     /**
+       * 
+       * Select all of the jobs, and select for delete the job.
+       * @Route("job/job_delete")
+       * @return \Symfony\Component\HttpFoundation\RedirectResponse|Response
+     */
+    public function getJobsForDelete()
+    {
+         return $this->render('job/job_delete.html.twig', array(
+            'jobs' => $this->jobService->getAllJobs()
+          ));
+    }
+    
+    /**
+      * 
+      * Delete the job from database
+      * 
+      * @Route("job/job_delete/{job_name}")
+     */
+    public function deleteJob($job_name)
```

Comment:
You should use ID as it is unique and integers can be fetched much faster by the DB!